docs/_source/docs/stack_config.rst

```diff
@@ -37,6 +37,9 @@ Sceptre treats the template as CloudFormation, Jinja2 or Python depending on
 the template’s file extension. Note that the template filename may be different
 from the Stack config filename.
 
+.. note::
+  This key is deprecated in favor of the `template`_ key.
```

Comment:
Maybe we can make it more explicit by adding something like "will be removed in 3.0.0".

Reply:
done